### ISSUE TITLE
clamav fails

### ISSUE THREAD

**Issue by fcorneli** [Homebrew/homebrew-core#113442]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 3.6.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: 978a452ea7ccdff1ed7377f88add3ff0513ba2b7
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: af0056b4243b00dff46f291d0db35180a4729508
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 20
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 20-core 64-bit cometlake
Clang: 14.0.0 build 1400
Git: 2.38.0 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6-x86_64
CLT: 14.0.0.0.1.1661618636
Xcode: 14.0.1
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Trying to start clamav in foreground.

### What happened (include all command output)?

```
/usr/local/Cellar/clamav/0.105.1/sbin/clamd --foreground --debug
LibClamAV Error: cl_load(): No such file or directory: /usr/local/Cellar/clamav/0.105.1/share/clamav
ERROR: Can't get file status
```

### What did you expect to happen?

That clamav starts.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
/usr/local/Cellar/clamav/0.105.1/sbin/clamd --foreground --debug
```

**Comment by fcorneli** [comment]
@gromgit Running `freshclam` indeed resolved the issue. I forgot that I did run this on a previous version. Thanks for the info.

**Comment by coreygo** [comment]
@gromgit I recently did a `brew upgrade` to version `0.105.1_1` and `freshclam` stopped working for me; `clamscan` and `freshclam` had been working without issue:

```
$ freshclam -v
Creating missing database directory: /usr/local/var/lib/clamav
ERROR: Failed to create database directory: /usr/local/var/lib/clamav
Manually prepare the database directory, or re-run freshclam with higher privileges.
ERROR: Initialization error!
``` 

I tried both `freshclam -v` and `sudo freshclam -v` but received that same error.

I also uninstalled and cleaned up all of the existing install before brew installing clamav again. I took care of the configs in `/usr/local/etc/clamav/` (only commented out the example line for first time set up) and tested again:

`freshclam` still fails with the same error and `clamconf -n` confirms `Database directory: /usr/local/var/lib/clamav`, everything else is okay otherwise.

I don't recall having to set the database directory previously… a change since #113485?
